src/transformers/__init__.py

```diff
@@ -937,8 +937,8 @@
 
 if is_flax_available():
     from .models.auto import FLAX_MODEL_MAPPING, FlaxAutoModel
-    from .models.bert import FlaxBertForMaskedLM, FlaxBertModel
-    from .models.roberta import FlaxRobertaModel
+    from .models.bert import FlaxBertForMaskedLM, FlaxBertForMaskedLMModule, FlaxBertModel, FlaxBertModule
+    from .models.roberta import FlaxRobertaModel, FlaxRobertaModule
```

Comment:
I have a concern in terms of scaling with having two objects per model in the main init. Is this really necessary?

Reply:
Good point and I agree with you! I think we don't need any of the `...Module` classes in the init - what do you think @mfuntowicz ?

Reply:
Will remove all ...Module from the inits - let me know what you think :-)

Reply:
Cool! They can be in the intermediate init (models.bert.__init__) if you think people might need them but it doesn't seem used anymore (except in the test that checks every model has its module).